src/main/java/com/conveyal/r5/analyst/cluster/AnalystWorker.java

```diff
@@ -238,10 +232,6 @@ public AnalystWorker(Properties config, TransportNetworkCache cache) {
         // http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/java-dg-roles.html
         s3 = new AmazonS3Client();
```

Comment:
I added code to set the region on the s3 object and modified how the default is set, so the default will be set both when running outside EC2 and when running offline (though not sure we'd ever need it when running offline).